.github/README.md

```diff
@@ -34,14 +34,25 @@ If you shutdown Docker and want to fire up the environment later, use `yarn env:
 
 ### Enable X-Debug
 
-To enable X-Debug in the containers, you need to set the environmental variables before starting up Docker:
-
+To enable X-Debug in the containers. You need to set the environmental variables before starting up Docker:
 * `export LOCAL_PHP_XDEBUG=true`
 * `export XDEBUG_HOST_IP=10.0.0.1`
+NOTE: change the IP `10.0.0.1` to the value of`$_SERVER[‘REMOTE_ADDR’]` when you load up the Docker-powered site.
+
+For Ubuntu users, run the command `ip a` and find the `docker0` alias. Copy the inet value.
+e.g. `inet 172.17.0.1/16`, use the value `172.17.0.1`.
+
+If you’ve brought up the containers already, run `yarn env stop` and `yarn env start` to reboot after setting the variable.
+
+- NOTE: For Ubuntu users: You need to include these variables every time you run `yarn env start` or `sudo yarn env start`.
```

Comment:
You only need to include the variables if you have to use `sudo`. This is because the current environmental variables set with `export` aren't forwarded to the `sudo` process. Again, this if for Linux users and not just Ubuntu users.